doc/user-guide.md

```diff
@@ -478,6 +479,47 @@ func main() {
 }
 ```
 
+##### If 3rd party resource does not have `AddToScheme()` function
+
+Use [SchemeBuilder][scheme_builder] from `controller-runtime` package.
```

Comment:
Nit: Just rephrasing this a bit since controller-runtime is not the package.
```suggestion
Use the [SchemeBuilder][scheme_builder] package from controller-runtime to initialize a new scheme builder that can be used to register the 3rd party resource with the manager's scheme.
```

Reply:
@hasbro17 sure, rephrased as you suggested, but I have to mention that https://godoc.org/sigs.k8s.io/controller-runtime claims that it's a package :)

Reply:
@ytsarev You're right actually :P I was thinking more specifically about the SchemeBuilder package but I forgot that everything is a package in Go.